### PR TITLE
Lambda inside vpc

### PR DESCRIPTION
**Pull request by pethers**
part of https://github.com/stelligent/cfn_nag/issues/503

### CODE REVIEW

**Comment by pethers** [comment]
Can anyone please review this. @arothian

**Comment by pethers** [comment]
Can somebody please review this !
@thegonch @oshaughnessy

File: lib/cfn-nag/custom_rules/LambdaFunctionInsideVPCRule.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require_relative 'base'
+
+class LambdaFunctionInsideVPCRule < BaseRule
+  def rule_text
+    'Lambda functions should be deployed inside a VPC, miss VpcConfig property'
```
Comment (arothian) [inline lib/cfn-nag/custom_rules/LambdaFunctionInsideVPCRule.rb:8]:
```suggestion
    'Lambda functions should be deployed inside a VPC'
```

File: spec/cfn_nag_integration/cfn_nag_lambda_permission_spec.rb [hunk 2]
```diff
@@ -33,6 +33,11 @@
                             message: 'IAM role should not allow * resource on its permissions policy',
                             logical_resource_ids: %w[LambdaExecutionRole],
                             line_numbers: [49]),
+              Violation.new(id: 'W89',
+                            type: Violation::WARNING,
+                            message: 'Lambda functions should be deployed inside a VPC, miss VpcConfig property',
```
Comment (arothian) [inline spec/cfn_nag_integration/cfn_nag_lambda_permission_spec.rb:38]:
```suggestion
                            message: 'Lambda functions should be deployed inside a VPC',
```

**Review by arothian — CHANGES_REQUESTED** [review]
Thanks @pethers for the contribution. I added some minor suggestions on rule text/description but this looks good. Thanks!

**Review by arothian — APPROVED** [review]

**Pull request merged**